src/Umbraco.Tests/Persistence/Repositories/PartialViewRepositoryTests.cs

```diff
@@ -22,7 +22,7 @@ public override void SetUp()
         {
             base.SetUp();
 
-            _fileSystem = new PhysicalFileSystem(Constants.SystemDirectories.MvcViews + "/Partials/", IOHelper);
+            _fileSystem = new PhysicalFileSystem(Current.IOHelper, Current.Logger, Constants.SystemDirectories.MvcViews + "/Partials/");
```

Comment:
Since this inherits from `UmbracoTestBase`, this should just use `Logger` (i.e. `this.Logger`) and `IOHelper` (i.e. `this.IOHelper`)